azure/worker/bindings/http.py

```diff
@@ -142,10 +145,17 @@ def from_proto(cls, data: protos.TypedData, *,
                 f'unsupported HTTP body type from the incoming gRPC data: '
                 f'{body_rpc_type}')
 
+        headers = {}
+        if data.http.headers:
+            for header_name, header_typed_value in data.http.headers.items():
```

Comment:
add / update http tests to include request headers

Reply:
Headers are covered by the unit tests already.  Do you mean something specific?

Reply:
Yes. Test headers with different types for values

Reply:
Not sure I follow.  We test functions by making HTTP requests, and per spec the headers are always strings. Can you give an example please?

Reply:
Here is an example Keep-Alive: 300

Reply:
> Here is an example Keep-Alive: 300 instead of Keep-Alive: "300"

But these are the same thing per HTTP spec.  In fact, `requests` **requires** that the headers dictionary only contains strings.

Reply:
Yup. I think you are right. If the requirement for headers to be <string, string>, we should not be changing RpcHttp in proto file. Will get back to you on this.

Reply:
I have reverted the changes to proto file. Please update the PR just to pull the latest changes. No other code changes needed. Thanks!